packages/forge/blocks/perplexity/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@typebot.io/perplexity-block",
+  "version": "1.0.0",
+  "private": true,
+  "type": "module",
+  "exports": {
+    ".": "./src/index.ts",
+    "./schemas": "./src/schemas.ts",
+    "./shared/*": "./src/shared/*.ts"
+  },
+  "dependencies": {
+    "@typebot.io/ai": "workspace:*",
+    "@typebot.io/forge": "workspace:*",
+    "@typebot.io/lib": "workspace:*",
+    "@typebot.io/zod": "workspace:*",
+    "axios": "^1.7.9"
```

Comment:
For all AI providers we are using Vercel's AI SDK: https://sdk.vercel.ai/.

They have a Perplexity package: https://sdk.vercel.ai/providers/ai-sdk-providers/perplexity